widget-missing-in-kendoui-professional-package.md

```diff
@@ -0,0 +1,97 @@
+
+---
+title: Widgets from the Kendo UI Core/Professional Package Don't Load
+description: After installing Kendo UI via the Package Manager, widget XXXX doesn't work
+type: Troubleshooting
+page_title: Widgets from the Kendo UI Core/Professional Package Don't Load
+slug: widget-missing-in-kendoui-professional-package
+position: 0
+tags: kendo, package manager, widget, error
+ticketid: 1017633
+
+---
+
+## Environment 
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package Manager</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+  <tr>
+    <td>Code (AppBuilder) client</td>
+    <td>All</td>
+  </tr>
+</table>
+
+## Description
+After installing Kendo UI Core or Professional via the Manage Packages window, widgets from the respective bundle (e.g. Grid, Scheduler, DropDownList) still don't load. 
```

Comment:
Nope, the customer usually installs Kendo UI Professional via the Package Manager and expects to be able declare a Grid widget.